x-pack/plugins/cloud_security_posture/common/utils/rules_states.ts

```diff
@@ -29,3 +29,11 @@ export const buildMutedRulesFilter = (
 
   return mutedRulesFilterQuery;
 };
+
+export const buildRuleKey = (
+  benchmarkId: string,
+  benchmarkVersion: string,
+  ruleNumber?: string
```

Comment:
This optional value raises some questions for me, we need to make sure the API can function when calling it without the rule number. if the API can't handle that we will need to skip this call or something